<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeBtn</class>
 <widget class="QWidget" name="EmployeeBtn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>员工添加</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>13</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>13</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>用户名：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>用户姓名：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>用户密码：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>确认密码：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>性别：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>地址：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>联系电话：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>类型：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>科室：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>挂号费用：</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>13</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEditUName"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditName"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditPassword"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSurePassword"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxSex"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditAddress"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditPhone"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxUserType"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxSectionRoom"/>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditMoney"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnCommit">
            <property name="text">
             <string>提交</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnReset">
            <property name="text">
             <string>重置</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>个人简介</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditInformation"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
